app/src/main/res/layout/drawer_header.xml

```diff
@@ -1,50 +1,71 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    style="@style/AppTheme.DrawerHeaderBackground"
     android:layout_width="match_parent"
-    android:layout_height="172dp"
-    android:theme="@style/ThemeOverlay.AppCompat.Dark">
+    android:layout_height="wrap_content"
+    android:background="@color/color_primary"
```

Comment:
IMHO that navigation drawer header is a tad too blue.
Just a quick thought: Grayish background color with a "150 Jahre - culture of excellence" ribbon?

Reply:
I‘ve tried a purely white drawer and didn’t find it appealing. I imagine that will be the case with a grayish background too. And I’d love to not include any of these “150 Jahre” things...

Reply:
I think most drawers have some [flair](https://www.youtube.com/watch?v=_ChQK8j6so8) in the header, so I made something up.

I agree, the 150 thingies are too cheesy...
Maybe something like the [Uhrenturm](https://www.vt.bgu.tum.de/typo3temp/_processed_/csm_Uhrenturm_01_b3b26c6dd1.jpg)?